src/librustc_metadata/cstore_impl.rs

```diff
@@ -427,7 +422,7 @@ impl cstore::CStore {
     pub fn load_macro_untracked(&self, id: DefId, sess: &Session) -> LoadedMacro {
         let data = self.get_crate_data(id.krate);
         if let Some(ref proc_macros) = data.proc_macros {
-            return LoadedMacro::ProcMacro(proc_macros[id.index.to_proc_macro_index()].1.clone());
+            return LoadedMacro::ProcMacro(proc_macros[id.index.to_proc_macro_index()].ext.clone());
```

Comment:
@petrochenkov Does this need to be cached, or could the `ext` be loaded here?

Reply:
Resolver already has a cache for this (`macro_map` in `fn get_macro_by_def_id`), so `load_macro_untracked` can just load the extension from metadata.